src/interfaces.ts

```diff
@@ -78,6 +87,12 @@ interface UseCollectionResultBase<T> {
     sortingDescending?: boolean;
     selectedItems?: ReadonlyArray<T>;
     onSelectionChange?(event: CustomEventLike<SelectionChangeDetail<T>>): void;
+    expandableRows?: {
+      getItemChildren: (item: T) => T[];
+      getItemExpandable: (item: T) => boolean;
+      expandedItems: ReadonlyArray<T>;
```

Comment:
I see two differences here with the API proposal:
- `getItemExpandable` was `isItemExpandable` (naming)
- `expandedItems: ReadonlyArray<T>` was `isItemExpanded: (item: T) => boolean`

Reply:
Thanks, the decision to favour expandedItems over isItemExpanded has been taken separately, I updated the doc. I will update the getItemExpandable.